roles/download/defaults/main.yml

```diff
@@ -479,7 +479,7 @@ haproxy_image_tag: 2.2
 # Coredns version should be supported by corefile-migration (or at least work with)
 # bundle with kubeadm; if not 'basic' upgrade can sometimes fail
 coredns_version: "1.7.0"
-coredns_image_repo: "{{ docker_image_repo }}/coredns/coredns"
+coredns_image_repo: "{{ kube_image_repo }}/coredns"
```

Comment:
The official release repo for CoreDNS is [DockerHub](https://coredns.io/manual/toc/#installation ). I know kubeadm default is the one in gcr.io but I'm wondering what we should use, the official one or the kubeadm's.

Reply:
I guess it doesn't really matter, as long as that's part of their release process now that CoreDNS is kube default DNS stack